### ISSUE TITLE
Non decreasable loss

### ISSUE THREAD

**Issue by RashaMahmoudAhmed** [zylo117/Yet-Another-EfficientDet-Pytorch#609]
please I would like to ask you something about this efficientDet model , when we use our custom dataset after evaluate our matrices and train our model when we evaluate the loss ,we got non decreasable loss from epoch 23 and we suppose to make 50 epoch , in addition to this sample of our dataset have this yellowish color which get us a wrong detection

![Capture](https://user-images.githubusercontent.com/74054940/111824173-859a2d80-88ee-11eb-9b19-5e6b8015c040.JPG)
[
![Capture11](https://user-images.githubusercontent.com/74054940/111826167-0823ec80-88f1-11eb-9359-ba19d1c10825.JPG)

](url)

**Comment by zylo117** [comment]
it has nothing to do with detection. it's only bgr rgb issue during display
